### DISCUSSION TITLE
graph recommendations?

### DISCUSSION THREAD

**Discussion by meganewing** [sr320/course-fish546-2021#123] [Q&A] [answered]
I want to make a graph of my data that shows differences in expression for control vs. treatment for all genes where there was a significant difference between the two groups. I drew what I'd imagine a graph like this to look like below.

I'm very unfamiliar with creating plots or graphing outside of excel. I think I could make a table of all the significantly different genes and then put that into excel to do this, but don't know if that is the right route to take.

Has anyone created a graph like this before and could point me in the right directions as well as offer some advice (or share a tutorial)? 

![image](https://user-images.githubusercontent.com/66333757/110186561-8dc07c00-7dca-11eb-8e42-f75ba9e21ec7.png)

**Comment by sr320** [comment, marked as answer]
I love all these questions :)!

Here is a model based on paper - https://docs.google.com/document/d/1dKwKKFfLoBPAswzMlaJ7Fk5BKDAN6C4ldv-6xpWIr9s/edit

But I would suggest the simple volcano plots that DESeq2 will generate.
see https://github.com/fish546-2021/aidan-hematodinium/blob/main/projects/hematodinium_analysis/graphs/DESeq2_output/amb0_vs_amb2_indiv/res05_MAplot.png 

And the trinity pipeline you leveraged before has a nice walk through of generating figures using DEseq2 including the above plot
https://github.com/trinityrnaseq/trinityrnaseq/wiki/Trinity-Differential-Expression#running-differential-expression-analysis

an example of a heat map.
<img width="481" alt="image" src="https://user-images.githubusercontent.com/4442039/110190372-d599d000-7dd7-11eb-85fe-a4c6e154ebad.png">

**Reply by meganewing** [reply]
I was able to generate a heat map and volcano plot last night, but think they are a lot less intuitive when it comes to interpretation--especially since gene names or sequences aren't specified. But I'll take a look at the paper you linked!

**Reply by sr320** [reply]
It's a bit data dependent- that is how many genes of interest are there- but we often focus on gene ontology visualizations (eg Revigo) 

Here is another example where gene ontology info is visualized and there are heat maps with specific gene names https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0133053

**Comment by aspencoyle** [comment]
If you really want to visualize counts of expression for individual genes, the plotCounts() function (part of DESeq2 pkg) might be useful! It just graphs one gene per plot, so the usefulness is limited, but more or less does what you're describing!
